Project/Managerclass.cs

```diff
@@ -21,8 +21,9 @@ public Manager(string setLevel, int setCost, string setName, string setSex, floa
     }
     public Manager()
     {
-        level = "rookie";
-        cost = 0;
+        level = 1;
+        cost = 200;

```

Comment:
Adjust the costs/income of a manager. Right now you're using the "cost" variable of the Manager class for the calculation in the `updateMoney` function in the Shop class, so each manager would result in a net loss of money for the player.